smetrics/src/main/scala/com/evolutiongaming/smetrics/MeasureDuration.scala

```diff
@@ -26,7 +26,7 @@ object MeasureDuration {
   def apply[F[_]](implicit F: MeasureDuration[F]): MeasureDuration[F] = F
 
 
-  def fromClock[F[_] : FlatMap](clock: Clock[F]): MeasureDuration[F] = {
+  implicit def fromClock[F[_] : FlatMap](implicit clock: Clock[F]): MeasureDuration[F] = {
```

Comment:
I've read a book Practical FP in Scala and there Volpe mentioned that it's ok to have such utility 'algebras' as a context bound. So why not `F[_]: FlatMap: Clock`? 😎

Reply:
yes, why not `[F[_]: FlatMap: Clock]`?

Reply:
BTW thank you for making this implicit, I hated making it manually >_<

Reply:
it was an instance constructor with an explicit argument, but now yes, it becomes just context bounds